apps/web/src/lib/sorters/task-sorters.ts

```
@@ -0,0 +1,71 @@
+          comparison = 0;
+        } else if (!a.dueDate) {
+          comparison = 1; // Tasks without due dates go to end
+        } else if (!b.dueDate) {
+          comparison = -1;
+        } else {
+          comparison =
+            new Date(a.dueDate).getTime() - new Date(b.dueDate).getTime();
+        }
+        break;
+      case "priority":
+        const priorityOrder = { urgent: 0, high: 1, medium: 2, low: 3 };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)